### PR TITLE
Avail samples

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
Adds Sample values for Availabilites and OperatingReport where they were missing. Updating PercentItemType description, adding DecimalPercentItemType and decimal_percent unit.

### CODE REVIEW

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -17932,6 +17990,10 @@
         "IECTS63019_2019": {
           "label": "IEC TS 63109 Ed. 1",
           "description": "Photovoltaic power systems (PVPS) – Information model for availability"
+        },
+        "SPEOM_2021": {
+          "label": "SPE O&M Guide v5.0:2021",
+          "description": "SolarPower Europe Operations and Maintenance Best Practices Guidelines, ver 5.0, 2021\""
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:17996]:
```suggestion
          "description": "SolarPower Europe Operations and Maintenance Best Practices Guidelines, ver 5.0, 2021"
```

**Review by cwhanse — APPROVED** [review]

**Review by jrippingale — APPROVED** [review]

**Comment by kevinkirkl3y** [comment]
Closes issue #230

**Review by jrippingale — APPROVED** [review]

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -16997,7 +17055,7 @@
       }
     },
     "PercentItemType": {
-      "description": "Percent unit",
+      "description": "Percent unit, described as a value between 0 and 1.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:17058]:
```suggestion
      "description": "Percent as a fration, where 100% is entered as 1.0",
```

**Comment by kevinkirkl3y** [comment]
Pushback on the value between 0-1 from Technical Group. If it is a value representing a percentage, the consensus was that it should not be the percentage/100. If we want the values to be represented as a decimal value is should be a DecimalTypeItemType.

**Review by jrippingale — APPROVED** [review]

**Review by cwhanse — APPROVED** [review]

**Pull request merged**